server/graphql/v2/object/Host.js

```diff
@@ -646,11 +657,13 @@ export const Host = new GraphQLObjectType({
           const expenseAmountOverTime = async () => {
             const dateFrom = args.dateFrom ? moment(args.dateFrom).toISOString() : undefined;
             const dateTo = args.dateTo ? moment(args.dateTo).toISOString() : undefined;
+            const timeUnit = getTimeUnit(numberOfDays);
```

Comment:
What happens to `args.timeUnit`? Sounds to me like we should remove its `defaultValue`, do something like:

```suggestion
            const timeUnit = args.timeUnit || getTimeUnit(numberOfDays);
```

And clarify all that in the field description.

Reply:
I've changed this to make sure the default value is removed and made the changes. Thanks. 🙂